ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Storages.resource

```diff
@@ -9,7 +9,7 @@ Resource       Projects.resource
 ${STORAGE_SECTION_XP}=             xpath=//div[@id="cluster-storages"]
 ${STORAGE_NAME_INPUT_XP}=          xpath=//input[@name="create-new-storage-name"]
 ${STORAGE_DESCR_INPUT_XP}=         xpath=//textarea[@name="create-new-storage-description"]
-${STORAGE_SIZE_INPUT_XP}=         xpath=//div/input[@aria-label="Input"]
+${STORAGE_SIZE_INPUT_XP}=         xpath=//div/input[contains(@aria-label,"input") or contains(@aria-label,"Input")]
```

Comment:
can you try `text()[contains(lower-case(@aria-label),'input')]` ?
[doc link](https://www.w3.org/TR/xpath-functions/#func-lower-case)

Reply:
hm sorry I think I messed up the xpath in my previous comment, it should be `div/input[contains(lower-case(@aria-label),'input')]`. Anyway, i tried on my end and didn't work neither..nevermind